src/components/Nav/nav.jsx

```diff
@@ -1,62 +1,75 @@
 import { NavLink, useNavigate } from "react-router-dom";
 import "../../pages/Home/home.css";
 import { signOut } from "../../services/signoutService";
-import { useAuth } from "../../contexts/auth-context"
+import { useAuth } from "../../contexts/auth-context";
+import { useWishlist } from "../../contexts/wishlist-context";
+import { unstable_createPortal } from "react-dom";
 
 const Nav = () => {
-
   const navigate = useNavigate();
-  const {auth: {Authenticated}, setAuth} = useAuth();
-   
-  {/* destructuring setauth and auth and further destructuring Authenticated from auth */}
+  const {
+    auth: { Authenticated },
+    setAuth,
+  } = useAuth();
+
+  const {
+    state: { wishlist },
+  } = useWishlist();
+
+  {
+    /* destructuring setauth and auth and further destructuring Authenticated from auth */
+  }
   return (
     <nav className="navigation-container">
       <div className="nav-brand">
         <NavLink className="header-style" to="/">
           NOZAMA
+          <i className="fa-solid fa-arrow-left-long nav-logo"></i>
         </NavLink>
-        <i className="fa-solid fa-arrow-left-long"></i>
       </div>
 
       <div className="input-search">
         <input className="search" placeholder="Search" type="search" />
       </div>
 
       <ul className="nav-pills">
-        {!Authenticated && <button className="nav-item button">
-          <NavLink className="login" to="/login-page">
-            Login
-          </NavLink>
-        </button>}
-
+        {!Authenticated && (
+          <button className="nav-item button">
+            <NavLink className="login" to="/login-page">
+              Login
+            </NavLink>
+          </button>
+        )}
```

Comment:
You can check for authentication and show Login/Logout at the same place. You can use ternary to check for both conditions.